src/main.ts

```diff
@@ -157,7 +179,7 @@ async function main() {
     }
 
     if (hasErrors) {
-      core.setFailed("validation found errors :(");
+      core.setFailed("EIP Walidator found issues");
```

Comment:
It's stylized as `eipw`, not EIP Walidator.

```suggestion
      core.setFailed("eipw found issues");
```